tables/bengali.cti

```diff
@@ -235,3 +235,4 @@ before BengaliVowel always	\x09E1 	6-123-1	# BENGALI LETTER VOCALIC LL
 before BengaliVowel always	\x09E3 	6-123-1	# BENGALIC LETTER VOCALIC LL
 before BengaliVowel always	\x09F0 	1235-1	# BENGALI LETTER RA WITH MIDDLE DIAGONAL (ASSAMESE)
 before BengaliVowel always	\x09F1 	1236-1	# BENGALI LETTER RA WITH LOWER DIAGONAL =  BENGALI LETTER VA WITH LOWER DIAGONAL (1.0) ASSAMESE
+always \x0964 256	dari character, equals with english full stop character dots
```

Comment:
I find the comment a bit confusing. I would drop everything after "dari".

Reply:
Final with dari character comment related I choosed the first form after 256 dot combination the comment block (I writing dari character comment).